pyrolite/util/skl/transform.py

```diff
@@ -78,7 +78,10 @@ def __init__(self, **kwargs):
 
     def transform(self, X, *args, **kwargs):
         if isinstance(X, pd.DataFrame):
-            out = X.applymap(self.forward)
```

Comment:
As for above, given the transform functions are known (and I think should be true for any 'universal function' in `numpy` speak), I think this can just be `out = self.forward(X); ... ; out = self.inverse(Y)`.